core/src/test/kotlin/com/codescene/jetbrains/core/git/GitStatusParserTest.kt

```
@@ -0,0 +1,9 @@
+package com.codescene.jetbrains.core.git
+
+import org.junit.Test
+
+class GitStatusParserTest {
+    @Test
+    fun `git status parsing is now handled by git4idea APIs`() {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)